roles/nfs_ganesha/tasks/main.yml

```diff
@@ -23,16 +28,26 @@
   tags:
     - prerequisites
 
+- name: check if gluster_features_ganesha_masternode is set
+  fail:
+    msg: "Variable gluster_features_ganesha_masternode has to be defined"
+  when: gluster_features_ganesha_masternode is not defined
+
 # The base directory has to be one of the directories unique in /tmp/
 # Generated in the main.yml of the task
 - name: Copy the ssh keys to the local machine
   fetch: src=/var/lib/glusterd/nfs/{{item}}
-         dest="{{ ganesha_tmp_dir }}"
+         dest= "{{ ganesha_tmp_dir }}"
```

Comment:
@Sheersha-jain the patch looks but. But why did you add a space after `=' here? It will not run, it errors out.